### PR TITLE
Fix node version

### PR DESCRIPTION
**Pull request by IvanPizhenko**

### CODE REVIEW

**Comment by IvanPizhenko** [comment]
@tomholub small change needed here, please approve, and I will be able to fix workflows in other repos

**Comment by IvanPizhenko** [comment]
@DenBond7 or maybe you can do it a bit faster than Tom

File: action.yml [hunk 1]
```diff
@@ -8,5 +8,5 @@ inputs:
     description: 'domains exempt from scanning'
     required: false
 runs:
-  using: 'node14'
+  using: 'node12'
```
Comment (DenBond7) [inline action.yml:11]: Could you clarify why you use a lower version?

Comment (IvanPizhenko) [reply action.yml:11]: This is what actually Github supports. I was thinking it supports node14 and tried to update from 12 but then got an error, so just reverting this back.

Comment (DenBond7) [reply action.yml:11]: Got it. Thank you!

**Review by DenBond7 — APPROVED** [review]

**Pull request merged**